src/Planners/tests/TimeVaryingDCMPlannerTest.cpp

```diff
@@ -23,42 +21,36 @@ TEST_CASE("TimeVaryingDCMPlanner")
 
     ContactListMap contactListMap;
 
-    iDynTree::Transform leftTransform = iDynTree::Transform::Identity();
-    iDynTree::Position leftPosition;
-
-    leftPosition[0] = 0;
-    leftPosition[1] = -0.8;
-    leftPosition[2] = 0;
-    leftTransform.setPosition(leftPosition);
+    // left foot
+    // first footstep
+    manif::SE3d leftTransform = manif::SE3d::Identity();
+    leftTransform.coeffs().head<3>() << 0, -0.8, 0;
     REQUIRE(contactListMap["left"].addContact(leftTransform, 0.0, 1.0));
 
-    leftPosition[0] = 0.25;
-    leftPosition[2] = 0.2;
-    leftTransform.setPosition(leftPosition);
+    // second footstep
+    leftTransform.coeffs()[0] = 0.25;
+    leftTransform.coeffs()[2] = 0.2;
```

Comment:
Please add a comment here as to how the coeffs() is ordered for the SE3d object. It should be 'pos+quat'. If someone modifying the tests, mistake it to be something else, then this would result in problems.

Reply:
Typical issue 7800437 fixes a bug I introduced with this PR.

Reply:
Actually, I would prefer if there is an equivalent of ``setPosition``.

Reply:
Relevant issue https://github.com/artivis/manif/issues/160

Reply:
Thanks for linking the issue, I missed it.